src/.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+  "root": true,
+  "parser": "babel-eslint",
+  "env": {
+    "browser": true
+  },
+  "extends": [
+    "@mapbox/eslint-config-mapbox/react",
+    "@mapbox/eslint-config-mapbox/import",
+    "@mapbox/eslint-config-mapbox/promise",
+    "prettier"
+  ],
+  "rules": {
+    "prefer-const": "off",
+
+    "react/no-deprecated": "off",
+    "react/no-string-refs": "off",
+    "react/no-access-state-in-setstate": "off",
+    "react/jsx-boolean-value": "off"
```

Comment:
Let's remove these `rules` exceptions. I put them in mapbox-react-components because it wasn't quite ready for all the eslint-config-mapbox rules and I figured we could gradually fix things. For this repo, we could start with all the rules.